src/main/java/uk/gov/hmcts/darts/event/model/DarNotifyApplicationEvent.java

```diff
@@ -8,10 +8,12 @@
 public class DarNotifyApplicationEvent extends ApplicationEvent {
     private final DartsEvent dartsEvent;
     private final DarNotifyType darNotifyType;
+    private final Integer courtroomId;
 
-    public DarNotifyApplicationEvent(Object source, DartsEvent dartsEvent, DarNotifyType darNotifyType) {
+    public DarNotifyApplicationEvent(Object source, DartsEvent dartsEvent, DarNotifyType darNotifyType, Integer courtroomId) {
```

Comment:
```suggestion
    public DarNotifyApplicationEvent(Object source, DartsEvent dartsEvent, DarNotifyType darNotifyType, Integer courtRoomId) {
```

Reply:
done, fixed in developer so that it could change all usages.
Although most usages in code are just courtroom.